apps/prairielearn/src/admin_queries/README.md

```diff
@@ -1,17 +1,32 @@
 # Administrator queries
 
-Each query must have two files:
+Each query must have the following files:
 
 - A JSON file in the [`adminQuery` JSON Schema](../schemas/schemas/adminQuery.json).
-- A SQL file with a query that returns a table, and potentially also performs an INSERT, UPDATE, or other action.
+- A JS or TS file with a default function that performs the necessary actions and returns the resulting data.
```

Comment:
Hopefully we can get rid of the need for a JSON file soonish? That is, I'd hope that we could declare the params, etc. in the TS file. Not requesting a change for this PR.

Reply:
Indeed, I considered that option, and I'm open to doing that in a future PR.